ui/components/app/modals/cancel-transaction/cancel-transaction-gas-fee/index.scss

```diff
@@ -1,6 +1,5 @@
 .cancel-transaction-gas-fee {
   background: #f1f4f9;
-  padding: 16px;
```

Comment:
🤔

Reply:
Oh I see there is another padding on line 6...

Reply:
Yup this is another one of these cases.